config/tests/jobs/presubmit-jobs.yaml

```diff
@@ -0,0 +1,158 @@
+# AUTOGENERATED by "UPDATE_FIXTURE_DATA=true go test ./config/tests/jobs". DO NOT EDIT!
```

Comment:
we should move this into a dedicated sub-folder so we can set a specific list of OWNERS for kubernetes presubmits

Reply:
Moved to `config/tests/jobs/policy` together with the test, a README.md and the OWNERS file. I copied that from ./config/testgrids/kubernetes/presubmits/OWNERS with some changes:
- removed emeritus
- added `no_parent_owners: true`
- added `sig/infra` as label

That leaves @BenTheElder, @aojea and @test-infra-oncall as reviewer/approver. I probably should remove the latter?